src/plugins/arkeo/LpCard.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Flex } from '@chakra-ui/react'
+import { useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { AssetIcon } from 'components/AssetIcon'
+import { Card } from 'components/Card/Card'
+import { Text } from 'components/Text'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import type { LpEarnOpportunityType } from 'state/slices/opportunitiesSlice/types'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { ArkeoCard } from './ArkeoCard'
+
+type LpCardProps = {
+  onClick: (arg: LpEarnOpportunityType) => void
```

Comment:
```suggestion
  onClick: (earnOpportunity: LpEarnOpportunityType) => void
```

Reply:
And same thing here, we should be able to only call `onClick` with an id, not the full structure